SBR.md

```diff
@@ -2049,8 +2049,8 @@ a. __Certificate Field:__ `subject:commonName` (OID 2.5.4.3)
 |---------|----------|
 | `Mailbox-validated` | Mailbox Address |
 | `Organization-validated` | `subject:organizationName` or Mailbox Address |
-| `Sponsor-validated` | Personal Name, `subject:pseudonym`, or Mailbox Address |
-| `Individual-validated` | Personal Name, `subject:pseudonym`, or Mailbox Address |
+| `Sponsor-validated` | Personal Name, Pseudonym, or Mailbox Address |
```

Comment:
Apologies for being so late to the party on this but I think this change has some unintended consequences which need addressing with further changes.

The previous version of this table was clear.  If the commonName contains a Pseudonym, then it must be a match for the subject:pseudonym.  I'm assuming this change is being made to remove the previously implied requirement that for subject:commonName to contain a pseudonym, subject:pseudonym must be provided?

However, I think this change opens the door for subject:commonName and subject:pseudonym containing _different_ Pseudonyms.  I've checked every reference to Pseudonyms in your version of the SMIME BRs and there is not (yet) any text which says that if the subject:commonName contains a pseudonym it must match any subject:pseudonym provided.  

As long as both of my Pseudonyms either uniquely identify me or uniquely identify me within the context of the subject:organizationName then I see no text in the document to prevent me from having two in a single certificate after this change.